packages/apex-node/test/execute/executeService.test.ts

```diff
@@ -29,8 +29,13 @@ describe('Apex Execute Tests', async () => {
 
   beforeEach(async () => {
     sandboxStub = createSandbox();
-    $$.setConfigStubContents('AuthInfoConfig', {
-      contents: await testData.getConfig()
+
+    $$.setConfigStubContents('GlobalInfo', {
+      contents: {
+        orgs: {
+          [testData.username]: await testData.getConfig()
+        }
+      }
```

Comment:
Most of the test updates were just converting how auth is mocked with the new core version.

Reply:
Mike has been adding to core3 to make this easier https://github.com/forcedotcom/sfdx-core/pull/600

Reply:
sweet.  Will include in improvements when we split this repo.